aspnetcore/blazor/performance/webassembly-event-pipe-diagnostics.md

```diff
@@ -41,10 +41,25 @@ The MSBuild properties in the following table enable profiler integration.
 
 Property | Default | Set value to&hellip; | Description
 --- | :---: | :---: | ---
-`<WasmPerfTracing>` | `false` | `true` | Controls diagnostic server tracing.
+`<WasmPerfTracing>` | `false` | `true` | Enables diagnostic server.
+`<WasmPerfInstrumentation>` | `none` | See table&dagger; | Enables instrumentation necessary for the sampling profiler. The property follows the :::no-loc text="callspec"::: syntax. &dagger;For permissible values, see the following table.
 `<MetricsSupport>` | `false` | `true` | Controls `System.Diagnostics.Metrics` support. For more information, see the [`System.Diagnostics.Metrics` namespace](/dotnet/api/system.diagnostics.metrics).
 `<EventSourceSupport>` | `false`| `true` | Controls `EventPipe` support. For more information, see [Diagnostics and instrumentation: Observability and telemetry](/dotnet/core/deploying/native-aot/diagnostics#observability-and-telemetry).
 
+The following table describes permissable `<WasmPerfInstrumentation>` values.
+
+`<WasmPerfInstrumentation>` value | Description
+--- | ---
+`all` | All assemblies
```

Comment:
```suggestion
`all` or `true` | All assemblies
```

Reply:
`true`/`all` and I'll need clarification on `false`/`none` to get the 😈 ironed out.